test/functional/cucumber/step_definitions/navigation.js

```diff
@@ -0,0 +1,40 @@
+/* jshint expr:true, -W079 */
+'use strict';
+
+var chai = require('chai'),
+    expect = chai.expect,
+    chaiAsPromised = require('chai-as-promised');
+
+chai.use(chaiAsPromised);
+
+module.exports = function() {
+    this.Then(/^I see a navigation wizard$/, function(next) {
+        expect(element(by.css('.step')).isPresent()).to.eventually.be.true.notify(next);
+    });
+
+    this.Then(/^I can reset my session by clicking on the 'Select File' Tab$/, function(next) {
+        element.all(by.css('.step-title')).filter(function(step) {
+            return step.getText().then(function(stepTitle) {
+                return (stepTitle === 'Select file & validate');
+            });
+        }).then(function(selectFileStep) {
+            selectFileStep[0].click();
+            expect(element(by.css('.btn__warning')).isPresent()).to.eventually.be.true.notify(next);
+        });
+    });
+
+    this.Then(/^I can navigate to the '([^']*)' page by clicking the '([^']*)' Tab$/, function(pageUrl, targetStepTitle, next) {
+        element.all(by.css('.step-title')).filter(function(step) {
+            return step.getText().then(function(stepTitle) {
+                return (stepTitle === targetStepTitle);
+            });
+        }).then(function(selectFileStep) {
+            selectFileStep[0].click();
+            browser.getCurrentUrl().then(function(url) {
+                var targetUrl = browser.baseUrl + '#/' + pageUrl;
```

Comment:
Now that @poorgeek has converted Angular to use HTML5 mode (see #403 and specifically, https://github.com/poorgeek/hmda-pilot/commit/5b8c0b2769698c94f1acf4e7093179ae79f4e3eb) the following `expect` will fail with the included `#` in the `targetUrl`